www/pages/index.tsx

```diff
@@ -28,45 +31,46 @@ export default function Home({ packages }) {
           <ul>
             {packages.map(({ name, version }) => (
               <li key="{name}-{version}">
-                <Link href={'/' + name}><a>{name}</a></Link> ({version})
+                <Link href={"/" + name}>
+                  <a>{name}</a>
+                </Link>{" "}
+                ({version})
```

Comment:
not entirely convinced we need the version thing here. The deployed site will be large enough even with just the latest versions. 🤷

Reply:
I think the design should accommodate wanting to include version numbers at some point. Even if it doesn't, it's useful to show what version the docs are *for*.